drivers/lora/sx1276.c

```diff
@@ -371,6 +375,8 @@ static int sx1276_lora_recv(struct device *dev, u8_t *data, u8_t size,
 	 * wise method to fix this!
 	 */
 	memcpy(data, dev_data.rx_buf, dev_data.rx_len);
+	memcpy(rssi, &dev_data.rx_rssi, sizeof(dev_data.rx_rssi));
```

Comment:
memcpy() for things smaller than u32_t ? 

Assign directly. But check for NULL first.

Reply:
Done